modules/exploits/linux/http/apache_superset_cookie_sig_rce.rb

```diff
@@ -0,0 +1,448 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache Superset Signed Cookie RCE',
+        'Description' => %q{
+          This exploit module illustrates how a vulnerability could be exploited
+          in a webapp.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die', # MSF module
+          'paradoxis', #  original flask-unsign tool
+          'Spencer McIntyre', # MSF flask-unsign library
+          'Naveen Sunkavally' # horizon3.ai writeup and cve discovery
+        ],
+        'References' => [
+          ['URL', 'https://github.com/Paradoxis/Flask-Unsign'],
+          ['URL', 'https://vulcan.io/blog/cve-2023-27524-in-apache-superset-what-you-need-to-know/'],
+          ['URL', 'https://www.horizon3.ai/cve-2023-27524-insecure-default-configuration-in-apache-superset-leads-to-remote-code-execution/'],
+          ['URL', 'https://www.horizon3.ai/apache-superset-part-ii-rce-credential-harvesting-and-more/'],
+          ['URL', 'https://github.com/horizon3ai/CVE-2023-27524/blob/main/CVE-2023-27524.py'],
+          ['EDB', '51447'],
+          ['CVE', '2023-27524'],
+          ['CVE', '2023-37941'] # rce
```

Comment:
Not a blocker; Do we ever reference other Metaploit modules here? 😄 

Edit: seems like we can use `RelatedModules`

https://github.com/rapid7/metasploit-framework/blob/c4396f2c6f45da68ec3fd8749a7d86fb54c2fa57/modules/auxiliary/scanner/http/emby_version_ssrf.rb#L20

We should probably update the other module with a reference to this new module too :+1:

Reply:
ironically that example is broken, `RelatedModules` has to be in the `Notes` block